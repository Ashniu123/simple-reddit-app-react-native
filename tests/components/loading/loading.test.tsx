import * as React from 'react';
import { shallow } from 'enzyme';

import Loading from '../../../src/components/loading/loading.component';

test('should render Loading component correctly', () => {
	const wrapper = shallow(<Loading />);
	
	expect(wrapper).toMatchSnapshot();
});