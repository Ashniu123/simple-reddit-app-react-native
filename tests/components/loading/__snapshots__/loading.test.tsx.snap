// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Loading component correctly 1`] = `
<ActivityIndicator
  animating={true}
  color="#0000ff"
  hidesWhenStopped={true}
  size="large"
/>
`;
